packages/commercetools/theme/nuxt.config.js

```diff
@@ -32,6 +32,9 @@ export default {
     '@nuxt/typescript-build',
     ['@vue-storefront/nuxt', {
       coreDevelopment: true,
+      logger: {
```

Comment:
`displayErrorsOnProd` will be valuable here as well

Reply:
I've introduces just verbosity `none` which means _don't log anything_ displaying errors etc. is depending on the logger you provide, eg. you are using sentry - it's your implementation of what you want to display and what now. Verbosity level means _which functions will be called_ (debug, error, info, debug)

Reply:
yeah, i also realized you can just do `varbosity: isProd ? "error" : "all"`